src/Microsoft.ML.Data/Transforms/NormalizeUtils.cs

```diff
@@ -70,27 +70,6 @@ internal interface IColumnFunction : ICanSaveModel
         NormalizingTransformer.NormalizerModelParametersBase GetNormalizerModelParams();
     }
 
-    internal static class NormalizeUtils
```

Comment:
>internal static class NormalizeUtils [](start = 4, length = 36)

double-checking that removing this is intentional.

Reply:
It is. It is an artifact of a time back in summer, back when we were first formulating ideas of what the public API would look like, and in fact the ideas around this were fairly ill formed. (This particular code was introduced in #433.) Clearly our thinking has moved on since then. This *particular* method (which as you see used to operate over `ColumnInfo`) is now directly taken care of by the `bool IsNormalized(this Schema.Column column)` extension method. And, because `RoleMappedSchema` now directly returns `Schema.Column` instead of `ColumnInfo` (which is the point of this PR, after all), there was no longer any reason to have this method. But thanks for checking @sfilipi.

---
In reply to: [243412114](https://github.com/dotnet/machinelearning/pull/1924#discussion_r243412114) [](ancestors = 243412114)